server/controllers/chartsController/testUtils.js

```diff
@@ -4,4 +4,18 @@ function postProcessDataFactory(mapEntries) {
   return new Map(Object.entries(mapEntries))
 }
 
-export { postProcessDataFactory }
+function variableCountsFromSubjectAssessmentDataFactory(overrides = []) {
+  const subjectAssessmentData = [
+    { day: 1, sex: 'male' },
+    { day: 2, sex: 'male' },
+    { day: 3, sex: 'male' },
+    { day: 4, sex: 'female' },
+    ...overrides,
+  ]
+  return subjectAssessmentData
+}
```

Comment:
this is a very specific factory since it deals with the "sex" attribute, and therefore not very extensible. Since it is only used in the charts controller helper tests, what do you think about defining

```ts
  const subjectAssessmentData = [
    { day: 1, sex: 'male' },
    { day: 2, sex: 'male' },
    { day: 3, sex: 'male' },
    { day: 4, sex: 'female' },
  ]
```

as a constant in that file and removing this factory?

Reply:
👍 
https://github.com/AMP-SCZ/dpdash/pull/260/commits/47f214a9fd64edcfca08b4b6b94fa57f4b13ff12